test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1482,16 +1483,15 @@ def revert_mom(self, mom, force=False):
                 # when the natural node is created below
                 # HUP may not be enough if exechost_startup is delayed
                 restart = True
+                enabled_cpuset = True
         if restart:
             mom.restart()
         else:
             mom.signal('-HUP')
         if not mom.isUp():
             self.logger.error('mom ' + mom.shortname + ' is down after revert')
-        if not mom.is_cpuset_mom():
-            self.server.manager(MGR_CMD_CREATE, NODE, None, mom.shortname)
         a = {'state': 'free'}
-        if mom.is_cpuset_mom():
+        if enabled_cpuset:
```

Comment:
Why have the enabled_cpuset variable if it is set when mom.is_cpuset_mom() is set.  Just use mom.is_cpuset_mom() like before.

Reply:
It's because it could be a mom.is_cpuset_mom() but mom.revert_to_default = False. I want to do the action specifically when enable_cpuset was done, where pbs_cgroups hook got enabled along with a default hook config file.

Reply:
I see it now, I thought it was set to True only if is_cpuset_mom() is true.  That if block is also in the if revert_to_defaults block.  Thanks for the explanation.